README.md

```diff
@@ -12,97 +12,26 @@ OSTIS Web platform contains:
 
 To learn more about the platform, check out our [documentation](https://github.com/ostis-ai/ostis-web-platform/blob/develop/docs/main.pdf).
 
-## Installation
+## Installation and usage
 
-- Quick start using Docker Compose
-
-  <details>
-  
-  <summary><b>Additional steps for Windows users</b></summary>
-
-  Make sure you are using UNIX line endings inside the repository and `longpaths` are enabled, otherwise you may face problems during build or installation process. Use the commands below to reconfigure Git on your machine:
-
-    ```sh
-    git config --global core.autocrlf input
-    git config --global core.longpaths true
-    ```
-
-  </details>
-
-  Requirements: you will need [Docker](https://docs.docker.com/get-docker/) installed and running on your machine.
+- Natively
 
   ```sh
-  git clone https://github.com/ostis-ai/ostis-web-platform
+  git clone (ostis-web-platform link)
   cd ostis-web-platform
-  # download images from Docker Hub
-  docker compose pull
-  # download KB and LaTeX docs plugin
-  git submodule update --init --recursive
-  # build knowledge base
-  docker compose run machine build
-  # launch web platform stack
-  docker compose up
-  ```
-
-   <details>
-   <summary> Building docker images locally </summary>
-
-  This may come in handy e.g. when you want to use a custom branch of the sc-machine or sc-web.
-
-  ### Requirements:
-
-  1. In case you're using Windows, set up git using the installation instructions above
-  2. Enable Docker BuildKit. You can use `DOCKER_BUILDKIT=1` shell variable for this.
-
-  ### Build process
-
-  ```sh
-  git clone https://github.com/ostis-ai/ostis-web-platform
-  git submodule update --init --recursive
+  git checkout feature/component_manager
   cd scripts
-  ./prepare.sh no_build_sc_machine no_build_sc_web # download all submodules without compilation.
-  cd ..
-  docker compose build
-  ```
-
-   </details>
-
-- Natively
-
-  Note: Currently, only Ubuntu is supported by this installation method. If you're going to use it, it might take a while to download dependencies and compile the components. Use it only if you know what you're doing!
-
-  ```sh
-  cd ostis-web-platform/scripts/
   ./prepare.sh
-  ```
-
-## Usage
-
-- Docker Compose
-
-  ```sh
-  # build the Knowledge Base.
-  # Required before the first startup (or if you've made updates to KB sources)
-  docker compose run machine build
-  # start platform services and run web interface at localhost:8000
-  docker compose up
-  ```
-
-- Native installation
-
-  ```sh
-  # Build knowledge base
-  cd ostis-web-platform/scripts/
+  cd ../sc-machine/scripts
```

Comment:
I think we should leave this docs about installation in docker and natively and write new docs about way of installation throught sc-component-manager. Totally, there must be three sections about installation